bin/foreman-proxy-certs-generate

```diff
@@ -21,13 +21,14 @@ class ForemanProxyCertsGenerate
   private
 
   def generate_answers_file
+    default_org = `cat /etc/foreman-installer/scenarios.d/katello-answers.yaml | grep initial_organization | cut -d '"' -f2`
```

Comment:
It'd be better to not shell out to do this, I think it'd be better to do [YAML.load_file](https://ruby-doc.org/stdlib-1.8.6/libdoc/yaml/rdoc/YAML.html#method-c-load_file) and get the file out of the answers hash.